javascript/javascript3/week1/giphySearch.js

```diff
@@ -0,0 +1,42 @@
+function btnGifClick() {
+  const searchKeyWord = document.getElementById("search-text").value;
+  let noOfGifs = document.getElementById("gif-count").value;
+
+  if (searchKeyWord !== "") {
+    if (
+      noOfGifs === "" ||
+      noOfGifs === null ||
+      noOfGifs === undefined ||
+      isNaN(noOfGifs)
+    )
+      noOfGifs = 5;
+    const url = new URL("https://api.giphy.com/v1/gifs/search");
+    const params = {
+      q: searchKeyWord,
+      limit: noOfGifs,
+      api_key: "pLg6OqlnCUK2JBmKNqWDJfHaFViJFvQ8",
+    };
+
+    Object.keys(params).forEach((key) => {
+      url.searchParams.append(key, params[key]);
+    });
+
+    fetch(url)
+      .then((response) => response.json())
+      .then((responseJson) => {
+        console.log(responseJson);
+        displaySearchResult(responseJson);
+      });
+  } else {
```

Comment:
This `else {} ` can work if the `noOfGifs` is a number.
And that comes from `input`.
So, it cannot be necessarily an empty `else {}`